cluster/internal/data.proto

```diff
@@ -23,7 +23,7 @@ message Tag {
 }
 
 message Point {
-	required string Name = 1;
+    required string Name = 1;
```

Comment:
To be clear, this is nothing to do with this change. I simply opened this file in my editor at one point, and it cleaned it up. I can revert.